kubernetes_asyncio/client/rest.py

```diff
@@ -142,7 +142,10 @@ async def request(self, method, url, query_params=None, headers=None,
 
         # For `POST`, `PUT`, `PATCH`, `OPTIONS`, `DELETE`
         if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
-            if re.search('json', headers['Content-Type'], re.IGNORECASE):
+            if (
```

Comment:
This file is autogenerated by the OpenAPI Generator, so to keep this change between releases we have to patch a file. Could you create patch for it, similar to this one deleted: [scripts/rest_client_patch.diff](https://github.com/tomplus/kubernetes_asyncio/pull/303/files#diff-5a785a7fc5b19b934975d4d49eb71d730de127362b35f89e368179b8355796c8)

Thanks.